### PR TITLE
Enable users to report trademark squatting via the "Report an Issue" page

### PR DESCRIPTION
**Pull request by HanilJain**
### Issue Solved 

#2236 

### Usage

when a user label a issue to "Trademark Squatting"
It loads other fields in form which are hidden 

image

In issue view it will be displayed in following way 

image

### CODE REVIEW

File: website/models.py [hunk 1]
```diff
@@ -274,6 +275,11 @@ class Issue(models.Model):
     reporter_ip_address = models.GenericIPAddressField(null=True, blank=True)
     cve_id = models.CharField(max_length=16, null=True, blank=True)
     cve_score = models.DecimalField(max_digits=2, decimal_places=1, null=True, blank=True)
+    infringing_domain_name = models.TextField(max_length=100, null=True, blank=True)
```
Comment (arkid15r) [inline website/models.py:278]: https://docs.djangoproject.com/en/5.0/ref/models/fields/#django.db.models.Field.null

**Review by arkid15r — CHANGES_REQUESTED** [review]
Please look into my suggestions when you get a chance:

**Review by DonnieBLT — CHANGES_REQUESTED** [review]
Please see @arkid15r 's comments

**Comment by HanilJain** [comment]
> Please see @arkid15r 's comments

sorry for the delay @DonnieBLT @arkid15r I'll do them today

File: website/models.py (hunk 1 above)
Comment (HanilJain) [reply website/models.py:278]: @arkid15r Sorry for the delay. thanks for guidance

File: website/views.py [hunk 2]
```diff
@@ -4607,7 +4629,7 @@ def AutoLabel(request):
         bug_description = data.get("BugDescription")
         template = """
         Label: {BugDescription}
-        Options: 0.General, 1.Number error, 2.Functional, 3.Performance, 4.Security, 5.Type, 6.Design, 7.Server down
+        Options: 0.General, 1.Number error, 2.Functional, 3.Performance, 4.Security, 5.Type, 6.Design, 7.Server down, 8.Tademark Sw
```
[the remark this answers is not in the document]
Comment (HanilJain) [reply website/views.py:4632]: corrected

[the remark this answers is not in the document]
Comment (HanilJain) [reply website/views.py:4632]: thanks @arkid15r

**Review by DonnieBLT — CHANGES_REQUESTED** [review]
no need to change the model, can you please update this

**Comment by HanilJain** [comment]
Closing this pull request due to a lot of merge conflicts.

**Pull request closed without merging**